RFC/src/RFC-0110_BaseNodes.md

```diff
@@ -115,18 +119,20 @@ The transaction validation service checks that:
 * The transaction does not have [timelocks] applied, limiting it from being mined and added to the blockchain before a
   specified block height or timestamp has been reached.
 * The transaction excess has a valid signature.
+* The [transaction weight] does not exceed the maximum permitted in a single block as defined by consensus.
 * The transaction excess is a valid public key. This proves that:
   $$ \Sigma \left( \mathrm{inputs} - \mathrm{outputs} - \mathrm{fees} \right) = 0 $$.
+* The [Tari script] of each input must execute successfully and return the public key that signs the script signature. 
+* The script offset \\( \so\\) is calculated and verified as per [RFC-0201_TariScript].
 
-`Rejected` transactions are dropped silently.
+Rejected transactions are dropped silently.
```

Comment:
The rejection is not strictly silent. The Base Node does tell the wallet about the rejection in the RPC response.